.github/workflows/build-test.yml

```
@@ -0,0 +1,27 @@
+on: [push, pull_request]
+name: Build and Test
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.23]
```

For libraries like this where it can become a burden to keep updating the Go version, I suggest using
```suggestion
        go-version: ['stable', 'oldstable']
```
This will always use the latest and the previous Go release.